examples/grpc_call/helloworld.rs

```diff
@@ -0,0 +1,410 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file is generated by rust-protobuf 2.22.1. Do not edit
```

Comment:
How was this generated? Could you include source `.proto` file and a way to regenerate this?

Reply:
Added `build.rs` to generate this code before build all.